kzg_prover/src/circuits/utils.rs

```diff
@@ -141,6 +141,7 @@ pub fn open_grand_sums<const N_CURRENCIES: usize>(
     advice_blinds: &[Blind<Fp>],
     params: &ParamsKZG<Bn256>,
     balance_column_range: Range<usize>,
+    polynomial_evaluations: &[Fp],
```

Comment:
Can you add the description of this new input in the comments? For all the apis that are now using `polynomial_evaluations` as input